cluster/src/main/java/com/hartwig/pipeline/SomaticPipeline.java

```diff
@@ -147,6 +147,7 @@ public PipelineState run(final AlignmentPair pair) {
                                     paveSomaticOutput,
                                     paveGermlineOutput,
                                     metadata.maybeTumor().map(t -> gripssSomaticProcessOutput).orElse(gripssGermlineProcessOutput),
```

Comment:
Are you sure this line is correct? Because in purple this argument is exclusively referred to as `gripssSomaticOutput`, while according to this line it could also be `gripssGermlineProcessOutput` (if `metadata.maybeTumor` is empty).

Reply:
Yes have to think about that - that line has been there for a while and doesn't relate to my change, since I pass the Gripss germline SV VCF as the next argument. It relates to running the pipeline in germline/reference-only mode.

Reply:
I've removed that condition since it should not have been there.